image_source_stream.go

```diff
@@ -2,8 +2,22 @@ package gostream
 
 import (
 	"context"
+	"errors"
+	"math"
+	"time"
 )
 
+// maxErrorSleepSec sets a maximum sleep time to the exponential backoff
+// determined by sleepTimeFromErrorCount
+var maxErrorSleepSec = math.Pow10(9) * 2 // two seconds in nanoseconds
```

Comment:
let's use https://pkg.go.dev/time#Duration here - it will make nanosec/millisec conversions easier

Reply:
nevermind, it's easier the way you did it